sample/src/main/java/com/mapzen/android/sample/RouteLineActivity.java

```diff
@@ -54,6 +54,10 @@
             return false;
           }
         });
+        ArrayList<LngLat> testPoints = new ArrayList();
+        testPoints.add(new LngLat(-122.39353246246766, 37.78662344689961));
+        testPoints.add(new LngLat(-122.39309926415683, 37.791273135641994));
+        map.drawTransitRouteLine(testPoints, "#ff0000");
```

Comment:
as long as this creates a data attribute (property) like `color: '#ff0000'` on a line geometry then my style will work (rather than relying on a scene file update).